buildenv/jenkins/JenkinsfileBase

```diff
@@ -1077,6 +1081,28 @@ def terminateTestProcesses() {
 	}
 }
 
+// For hide app windows to aid AWT and Arctic automated test execution
+def hideAppWindows() {
+        if (params.PLATFORM && params.PLATFORM.contains("mac")) {
```

Comment:
Please also add a check for `BUILD_LIST` to limit this to only required tests (i.e., openjdk).

Reply:
ah good plan, thanks

Reply:
done